conf/ssl4eo-l.yaml

```diff
@@ -0,0 +1,36 @@
+module:
+  _target_: torchgeo.trainers.MoCoTask
+  model: resnet50
+  weights: True
```

Comment:
Do we want to start from ImageNet weights?

Reply:
`True` should use imagenet weights. See [link](https://github.com/microsoft/torchgeo/blob/3196dab4dbb9df15d47f266e0fa2c7ec6eb415a3/torchgeo/trainers/moco.py#L227)